static/app/components/feedback/feedbackItem/feedbackScreenshot.tsx

```diff
@@ -28,23 +30,29 @@ export default function FeedbackScreenshot({
         if (!hasRole) {
           return null;
         }
+
+        const img = (
+          <StyledImageVisualization
```

Comment:
<img width="377" alt="SCR-20240402-pdth" src="https://github.com/getsentry/sentry/assets/56095982/66a71757-22c5-49d6-95cb-b60d6693311e">

nit: could we add `border-radius: 2px` here? i looked too closely at @Jesse-Box's [figma](https://www.figma.com/file/hIUACYgu3lenuAPLgg22a3/Specs%3A-Screenshots-v1?type=design&node-id=30-5771&mode=design&t=IeaQ0foqkLdEyqPp-0)